backend/src/db/migrations/20230920200620_record_achieved_achievements_in_users_achievements_table.ts

```diff
@@ -0,0 +1,38 @@
+import { type Knex } from 'knex';
+
+const TableName = {
+  USERS_ACHIEVEMENTS: 'users_achievements',
+  ACHIEVEMENTS: 'achievements',
+  ARTICLES: 'articles',
+  COMMENTS: 'comments',
+} as const;
+
+const up = (knex: Knex): Promise<void> => {
+  return knex.raw(`
+    WITH UsersCounts AS (
```

Comment:
As CTE is still structure like a table - let's call it as tables
```suggestion
    WITH user_counts AS (
```